### DISCUSSION TITLE
How to draw a stacked shape?

### DISCUSSION THREAD

**Discussion by soasada** [plantuml/plantuml#1267] [Q&A]
Hi, is it possible to draw something like this:

<img width="281" alt="image" src="https://user-images.githubusercontent.com/7634531/212658151-1df7b8c2-e668-4e9d-a2e4-01d4a9eefab4.png">

**Comment by The-Lum** [comment]
Hello @soasada, and all,

Depending of the diagram target, we can use `collections` _(without nested elements)_, as:
```puml
@startuml
collections collections
node node
@enduml
```
>[![](https://www.planttext.com/api/plantuml/svg/SoWkIImgAStDuKhEpyb9JIx9pCyhLa1YS-NbfwGgW0WkXzIy5A1A0000)](https://www.planttext.com/?text=SoWkIImgAStDuKhEpyb9JIx9pCyhLa1YS-NbfwGgW0WkXzIy5A1A0000)

_If that can help,_
Regards.

**Reply by soasada** [reply]
Is it possible to add nested elements to the collection?

**Reply by The-Lum** [reply]
Unfortunately not yet, see this wanted request:
- https://forum.plantuml.net/16964/could-collections-support-nested-elements
- https://forum.plantuml.net/14482/can-subcomponents-be-placed-inside-collections
